### ISSUE TITLE
Move security and login related Behat tests back into framework

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-framework#8256]
## Affected Version

4.x

## Description

Behat tests were moved from framework to admin in #8112, but we're now seeing test failures in silverstripe/admin that are caused by and testing changes in framework. I vote for moving the login and security related tests back into framework from admin.

**Comment by chillu** [comment]
I don't think this is a question of concurrency and Travis slowness, it's where the functionality lives which Behat tests. The login from logic is contained in `silverstripe/framework`, not `silverstripe/admin`, so tests related to it should also live there.

**Comment by robbieaverill** [comment]
Yeah exactly. Also testing features (like login/logout) that you get in a project that doesn't have the admin module installed.

**Comment by unclecheese** [comment]
Yeah that makes sense. The whole authentication workflow is a feature of framework, not admin. As long as "security tests" excludes SecurityAdmin, I'm fine with this. The original migration of the framework behat tests was probably a bit too greedy.
